contentcuration/contentcuration/urls.py

```diff
@@ -259,14 +462,7 @@ def get_queryset(self):
     url(r'^channels/(?P<channel_id>[^/]{32})/view', views.channel_view_only, name='channel_view_only'),
```

Comment:
RE redirect endpoints: this logic can probably be moved to the vue router (with the permission checks happening there as well). One motivation behind the `url(r'^channels/(?P<channel_id>[^/]{32})/?$', views.redirect_to_channel, name='redirect_to_channel')` endpoint is that you can send people a link to a channel without necessarily knowing if they're an editor or viewer. However, this can probably be accomplished in the vue router too

Reply:
Yeah - I think from our discussion, the unsuffixed URL could just be the URL with edit/view behaviour determined by user permissions and ricecooker status.